manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -16,6 +16,7 @@ gateways:
     labels:
       app: istio-egressgateway
       istio: egressgateway
+      security.istio.io/tlsMode: istio
```

Comment:
This doesn't seem safe? Won't we accidentally start doing autoMTLS to the gateway?